tests/filestore-v2.13-filedata/test.rules

```diff
@@ -0,0 +1 @@
+alert http any any -> any any (http.uri; content: "toto"; filestore; file.data; content: "tata"; sid:1;)
```

Comment:
Thanks for adding this, but I think my remark was about false positive (instead of false negative)

So, we want to check that `alert http any any -> any any (http.uri; content: "toto"; filestore; file.data; content: "titi"; sid:2;)` does not match